src/com/projectkorra/projectkorra/configuration/ConfigManager.java

```diff
@@ -1287,6 +1292,11 @@ public static void configCheck(final ConfigType type) {
 			config.addDefault("Abilities.Water.IceBullet.ShotCooldown", 500);
 			config.addDefault("Abilities.Water.IceBullet.Combination", Arrays.asList("WaterBubble:SNEAK_DOWN", "WaterBubble:SNEAK_UP", "IceBlast:SNEAK_DOWN"));
 
+			config.addDefault("Abilities.Water.PlantTether.Enabled", true);

```

Comment:
I think we will remove all these and PlantTether just for the release. Just since having them in the config is going to create people asking questions about it for now

(But I can do that, just stating my thoughts)

Reply:
Ill comment them out. They won't be out of commission for too long haha.